json_pure.gemspec

```
@@ -16,36 +16,17 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = ["README.md"]
   s.rdoc_options = ["--title", "JSON implementation for ruby", "--main", "README.md"]
+
   s.files = [
     "CHANGES.md",
     "COPYING",
     "BSDL",
     "LEGAL",
     "README.md",
-    "json_pure.gemspec",
-    "lib/json.rb",
-    "lib/json/add/bigdecimal.rb",
-    "lib/json/add/complex.rb",
-    "lib/json/add/core.rb",
-    "lib/json/add/date.rb",
-    "lib/json/add/date_time.rb",
-    "lib/json/add/exception.rb",
-    "lib/json/add/ostruct.rb",
-    "lib/json/add/range.rb",
-    "lib/json/add/rational.rb",
-    "lib/json/add/regexp.rb",
-    "lib/json/add/set.rb",
-    "lib/json/add/struct.rb",
-    "lib/json/add/symbol.rb",
-    "lib/json/add/time.rb",
-    "lib/json/common.rb",
-    "lib/json/ext.rb",
-    "lib/json/generic_object.rb",
-    "lib/json/pure.rb",
-    "lib/json/pure/generator.rb",
-    "lib/json/pure/parser.rb",
-    "lib/json/version.rb",
+    "json.gemspec",
```

```suggestion
    "json_pure.gemspec",
```
or not including it at all